avm-transpiler/src/transpile_contract.rs

```diff
@@ -3,52 +3,56 @@ use log::info;
 use serde::{Deserialize, Serialize};
 
 use acvm::acir::circuit::Program;
+use noirc_errors::debug_info::DebugInfo;
+use noirc_errors::debug_info::ProgramDebugInfo;
 
-use crate::transpile::brillig_to_avm;
+use crate::transpile::{brillig_to_avm, map_brillig_pcs_to_avm_pcs, patch_debug_info_pcs};
 use crate::utils::extract_brillig_from_acir_program;
 
 /// Representation of a contract with some transpiled functions
 #[derive(Debug, Serialize, Deserialize)]
-pub struct TranspiledContract {
+pub struct TranspiledContractArtifact {
     pub transpiled: bool,
     pub noir_version: String,
     pub name: String,
     // Functions can be ACIR or AVM
-    pub functions: Vec<AvmOrAcirContractFunction>,
+    pub functions: Vec<AvmOrAcirContractFunctionArtifact>,
```

Comment:
These were incorrectly named before. They are named after the ContractArtifact and ContractFunctionArtifact structs in Noir.